src/OptimalTransport.jl

```diff
@@ -715,33 +715,82 @@ Returns the entropically regularised barycenter of the `μ`, i.e. the histogram
 where ``\\operatorname{OT}_{ε}(\\mu, \\nu) = \\inf_{\\gamma \\Pi(\\mu, \\nu)} \\langle \\gamma, C \\rangle + \\varepsilon \\Omega(\\gamma)`` 
 is the entropic optimal transport loss with cost ``C`` and regularisation ``\\epsilon``.
 """
-function sinkhorn_barycenter(μ, C, ε, w; tol=1e-9, check_marginal_step=10, max_iter=1000)
-    sums = sum(μ; dims=1)
-    if !isapprox(extrema(sums)...)
-        throw(ArgumentError("Error: marginals are unbalanced"))
+function sinkhorn_barycenter(
+    μ,
+    C,
+    ε,
+    w;
+    tol=nothing,
+    atol=tol,
+    rtol=nothing,
+    check_marginal_step=nothing,
+    check_convergence=10,
+    max_iter=nothing,
+    maxiter=1_000,
```

Comment:
This should be
```suggestion
    max_iter=nothing,
    maxiter=max_iter,
```
otherwise `max_iter` is just ignored. And then one has to use
```julia
_maxiter = maxiter === nothing ? 1_000 : maxiter
```
in the iteration.